packages/app/components/Queue/TA/TAQueueDetailButtons.tsx

```diff
@@ -136,6 +142,21 @@ export default function TAQueueDetailButtons({
         return [true, "Help Student"];
       }
     })();
+    const [canRephrase, rephraseTooltip] = ((): [boolean, string] => {
```

Comment:
merged the two ternaries into this pair. kinda just followed the same pattern as `[canHelp, helpTooltip]`, but do lmk if there's a better way to organize them